core/block/detailservice/relations.go

```diff
@@ -97,6 +104,14 @@ func (s *service) ListRelationsWithValue(spaceId string, value *types.Value) ([]
 		}
 	}
 
+	if mentionsCounter > 0 {
```

Comment:
Maybe write custom sorter with `sort.Slice`? Mentions first, then all other relations alphabetically

Reply:
Yes, I was thinking about it, but it will increase calculations time and sorting looks lil bit ugly :see_no_evil: 

```
sort.Slice(keys, func(i, j int) bool {
		if keys[i] == bundle.RelationKeyMentions.String() {
			return true
		}
		if keys[j] == bundle.RelationKeyMentions.String() {
			return false
		}
		return keys[i] < keys[j]
	})
```